client/src/modules/cohort/model/actions.js

```diff
@@ -203,13 +203,15 @@ export const updateCohort = (cohortName, cohortId, data) => dispatch =>
 
 export const deleteCohort = (cohortId, cohortName) => dispatch =>
   deleteData(`/api/cohorts/${cohortId}/delete`)
-    .then(() => {
+    .then(resp => resp.json())
+    .then(members => {
```

Comment:
I know that it's confusing and unclear. Usually, we don't send data on deletion. But we emit socket events on the frontend. So if the cohort is deleted we want to send some info to the members of this cohort and redirect them from this cohort, etc. But socket event is emitted after the cohort is deleted so all cohort data is lost so we can't get this data and send an event to cohort members because we don't know them. So on cohort deletion, I send cohort's members ids to know who should I send events to. This problem disappears when we move emitting events to controllers and we will not have to do such strange things...